packages/app-cli/tests/services_KeymapService.js

```diff
@@ -242,7 +242,7 @@ describe('services_KeymapService', () => {
 		it('should throw when the required properties are missing', () => {
 			const customKeymaps = [
 				[
-					{ commmmmand: 'gotoAnything', accelerator: 'Ctrl+Shift+G' },
+					{ command: 'gotoAnything', accelerator: 'Ctrl+Shift+G' },
```

Comment:
I think this might have been on purpose. This is called a negative test. But I might be wrong. 

Laurent would know.

Reply:
Just saw you reverted that again.

Reply:
yeah saw it in the check logs. I thought I was fixing a typo.

Reply:
rather it was a test that was expecting an error.

Reply:
Yep, this is called a negative test.